feature/amazon/src/main/java/com/revenuecat/purchases/amazon/storeProductConversions.kt

```diff
@@ -98,29 +67,26 @@ internal data class Price(
 // The lasts two are englobed in []*, as they can be repeated 0 or n times.
 private val pattern: Pattern = Pattern.compile("(\\d+[[\\.,\\s]\\d+]*)")
 
-internal fun String.parsePriceAndCurrencySymbolUsingRegex(numberFormat: NumberFormat): Pair<Float, String>? {
+internal fun String.parsePriceUsingRegex(): Float? {
     val matcher = pattern.matcher(this)
     return if (matcher.find()) {
-        val price: String = matcher.group()
-        val currencySymbol = this.replace(price, "").trim()
-        val priceNumeric = extractPriceNumber(price, numberFormat)
-
-        priceNumeric to currencySymbol
-    } else null
-}
-
-private fun extractPriceNumber(
-    price: String,
-    numberFormat: NumberFormat
-): Float {
-    return price.trim().let { formattedPriceWithoutSymbol ->
-        try {
-            numberFormat.parse(formattedPriceWithoutSymbol).toFloat()
-        } catch (e: ParseException) {
-            warnLog(AmazonStrings.PRICE_EXTRACTION_PARSE_EXCEPTION.format(price))
-            0.0f
+        val dirtyPrice = matcher.group()
```

Comment:
is this also from unity? will this break for app tester, since they use periods for the 1,00 example?